README.md

```diff
@@ -54,32 +54,35 @@ Managed node host:
 * Docker >= 1.12
 * python >= 2.6
 * docker-py >= 1.7.0
+* Java => 8
 
 Make sure that the injector host’s hardware resources is not the performance tests bottleneck. Enough cpu, memory and network resources should be available.
 
-### Common settings
+*Installation Step*:
 
-** HTTP/HTTPS **
```

Comment:
Document how to setup HTTP or HTTPS traffic

Reply:
Will add your suggested changes.